### PR TITLE
add project backend

### PR DESCRIPTION
**Pull request by ywu385-c**

### CODE REVIEW

File: src/main/java/com/bytewizard/vsagilebackend/service/impl/TbProjectServiceImpl.java [hunk 1]
```diff
@@ -20,6 +25,27 @@ public class TbProjectServiceImpl extends ServiceImpl<TbProjectMapper, TbProject
     public TbProjectServiceImpl(TbProjectMapper tbProjectMapper) {
         this.tbProjectMapper = tbProjectMapper;
     }
+
+    @Override
+    public Integer createProject(ProjectDTO projectDTO) {
```
Comment (zyg0121) [inline src/main/java/com/bytewizard/vsagilebackend/service/impl/TbProjectServiceImpl.java:30]: pls tell me whether you already tested this fun.

Comment (zyg0121) [reply src/main/java/com/bytewizard/vsagilebackend/service/impl/TbProjectServiceImpl.java:30]: And give the result of the db.

**Review by zyg0121 — COMMENTED** [review]
pls answer these question.

File: src/main/java/com/bytewizard/vsagilebackend/service/impl/TbProjectServiceImpl.java (hunk 1 above)
Comment (ywu385-c) [reply src/main/java/com/bytewizard/vsagilebackend/service/impl/TbProjectServiceImpl.java:30]: Yes, the result shows :{
      
      "name": "Julie Conn",
      "pm_id": 100,
      "status": 1,
      "description": "Accusamus somniculosus sonitus. Fuga audio aveho cruentus agnosco turbo speciosus peccatus cornu uredo. Tardus adimpleo cupiditate.",
      "start_date": "2025-08-13",
      "end_date": "2024-02-16"
    }

**Review by zyg0121 — APPROVED** [review]
agree

**Pull request merged**